src/modules/recipes/NewRecipeForm/CreateRecipe.tsx

```diff
@@ -36,16 +38,24 @@ export const CreateNewRecipe = () => {
 
     formState: { errors },
   } = useForm({
-    strMeal: "",
-    strCategory: "",
-    ingredients: [{ ingredient: "", measure: "" }],
+    defaultValues: {
+      strMeal: "",
+      strCategory: "",
+      strArea: "",
+      strInstructions: "",
+      strDrinkAlternate: "",
+      strMealThumb: "",
+      strTags: "",
+      strYoutube: "",
+      ingredients: [{ ingredient: "", measure: "" }],
+    },
   });
 
   const handleSnackbarClose = () => {
     setSnackbarOpen(false); // Close Snackbar
   };
 
-  const onSubmit = (data) => {
+  const onSubmit = (data: any) => {
```

Comment:
Instead of using any here, add an interface for the defaultValues used in useForm hook. Something like:
```
interface NewRecipeFormData {
  strMeal: string;
  strCategory: Category;
  // and so on
}
```